Game.java

```diff
@@ -1,20 +1,20 @@
-import java.io.BufferedReader;
-import java.io.InputStream;
-import java.io.InputStreamReader;
+import java.io.*;
 import java.util.Scanner;
 
 public class Game {
 
     private InputStream stream;
     private Board board;
-    private String gameMode = "pvp";
+    public String gameMode = "pvp";
     private Player playerActive;
     private Player playerDeactive;
     private BufferedReader inputReader;
+    private GameType gameType;
 
-    public Game(InputStream stream, Board board) {
+    public Game(InputStream stream, Board board, GameType gameType) {
         this.stream = stream;
         this.board = board;
+        this.gameType = gameType;
```

Comment:
gameType is a weird name as it conjures up PVP, PVC, CVC.. 
gameType here is the dependency responsible for the user interaction...